### PR TITLE
Fix for pronouns without grmClass

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/alpheios-core/issues/209

small fix for creating InlfectionData and the test for it

I made changes only in `inflection-tables` packages, all other changes are from `es-lint`

### CODE REVIEW

File: packages/client-adapters/src/adapters/lexicons/adapter.js [hunk 1]
```diff
@@ -369,28 +369,34 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
     // if we still don't have a match, we can do a last ditch check without
     // any diacritics at all in those languages that support it
     if (!found) {
-      let lastAlt = []
+      const lastAlt = []
```
Comment (balmas) [inline packages/client-adapters/src/adapters/lexicons/adapter.js:372]: I can never remember, but don't we need to define arrays with let if new values are going to be pushed to them?

Comment (balmas) [reply packages/client-adapters/src/adapters/lexicons/adapter.js:372]: (if so, this is my mistake for not including the necessary eslint here I know...)

Comment (irina060981) [reply packages/client-adapters/src/adapters/lexicons/adapter.js:372]: yes - here should be special es-lint commands
I didn't touch this package during development - that's why I decided not to fix it bfore review - I would add this es-lint rule

Comment (balmas) [reply packages/client-adapters/src/adapters/lexicons/adapter.js]: thank you!

**Pull request merged**